### ISSUE TITLE
[Docker image] grimoirelab/full: manuscript - latex packages missing

### ISSUE THREAD

**Issue by ioiobzit** [chaoss/grimoirelab#289]
Hello,

during the generation of reports, manuscript gives following error:
```
root@grimoire:~# manuscripts -d /reports -u http://localhost:9200 -n '[REDACTED WITH SPACES]' --data-sources git
2020-02-06 09:58:45,957 GET http://localhost:9200/git/_search [status:200 request:0.011s]
2020-02-06 09:58:45,958 Generating the report from 2010-09-22 00:00:00+00:00 to 2020-02-05 23:59:59.999999+00:00
2020-02-06 09:58:45,959 Generating the report data and figs from 2010-09-22 00:00:00+00:00 to 2020-02-05 23:59:59.999999+00:00
2020-02-06 09:58:45,959 Generating Overview
2020-02-06 09:58:46,058 GET http://localhost:9200/git/_search [status:200 request:0.097s]
2020-02-06 09:58:46,295 GET http://localhost:9200/git/_search [status:200 request:0.234s]
2020-02-06 09:58:47,938 Generating Communication Channels
2020-02-06 09:58:47,938 Generating Detailed Activity by Project
2020-02-06 09:58:47,938 Activity data for: general
2020-02-06 09:58:48,000 GET http://localhost:9200/git/_search [status:200 request:0.060s]
2020-02-06 09:58:48,059 GET http://localhost:9200/git/_search [status:200 request:0.057s]
2020-02-06 09:58:49,798 Community data for: general
2020-02-06 09:58:49,857 GET http://localhost:9200/git/_search [status:200 request:0.056s]
2020-02-06 09:58:51,464 GET http://localhost:9200/git/_search [status:200 request:0.264s]
2020-02-06 09:58:51,476 GET http://localhost:9200/git/_search [status:200 request:0.009s]
2020-02-06 09:58:51,477 Process data for: general
2020-02-06 09:58:51,478 Data and figs done
2020-02-06 09:58:51,478 Generating PDF report
/bin/sh: 1: pdflatex: not found
2020-02-06 09:58:51,748 Error generating PDF
2020-02-06 09:58:51,750 Report completed
```

To make sure that the reports could be exported correctly the following list of packages have been installed manually:
* texlive-latex-base
* texlive-latex-extra
* latex-cjk-common

How to install them:
1. enter the docker container
```docker exec -it CONTAINERNAME /bin/bash```
2. install packages
```
$ sudo su -
# apt update
# apt install texlive-latex-base latex-cjk-common texlive-latex-extra
```

If possible please integrate them directly in the grimoirelab/full docker container.
Thank you.

**Comment by valeriocos** [comment]
Hi @ioiobzit , I have prepared a PR to include these packages in `docker/Dockerfile-installed`. If I understood correctly, these packages should be included in that image (https://github.com/chaoss/grimoirelab/tree/master/docker#grimoirelabinstalled). Grimoirelab/full inherits from the installed one (https://github.com/chaoss/grimoirelab/blob/master/docker/Dockerfile-full#L17)

**Comment by valeriocos** [comment]
Fixed with #291
